README.md

```diff
@@ -370,7 +370,7 @@ cabal exec cardano-cli -- byron create-byron-proposal-vote
 Byron vote submission:
 
 ```
-cabal exec cardano-cli -- byron submit-byron-proposal-vote
+cabal exec cardano-cli -- byron submit-proposal-vote
```

Comment:
As above, a `node` part needs to be added: `cabal exec cardano-cli -- byron node submit-proposal-vote`